App.tsx

```diff
@@ -1,35 +1,37 @@
 import React, { FunctionComponent, useState, useEffect } from "react"
-import SplashScreen from "react-native-splash-screen"
 import "array-flat-polyfill"
 import env from "react-native-config"
+import SplashScreen from "react-native-splash-screen"
 
 import MainNavigator from "./src/navigation/MainNavigator"
 import { ErrorBoundary } from "./src/ErrorBoundaries"
 import { ExposureProvider } from "./src/ExposureContext"
-import {
-  OnboardingProvider,
-  onboardingHasBeenCompleted,
-} from "./src/OnboardingContext"
+import { OnboardingProvider } from "./src/OnboardingContext"
 import { ConfigurationProvider } from "./src/ConfigurationContext"
 import { PermissionsProvider } from "./src/PermissionsContext"
 import { SystemServicesProvider } from "./src/SystemServicesContext"
 import { initializei18next, loadUserLocale } from "./src/locales/languages"
+import { StorageUtils } from "./src/utils"
 import Logger from "./src/logger"
 
 Logger.start()
 
 const App: FunctionComponent = () => {
   const [isLoading, setIsLoading] = useState(true)
-  const [onboardingIsComplete, setOnboardingIsComplete] = useState(false)
+  const [isOnboardingComplete, setIsOnboardingComplete] = useState(true)
+
+  const determineIsOnboardingComplete = async (): Promise<boolean> => {
+    return await StorageUtils.getIsOnboardingComplete()
```

Comment:
Curious to learn why we moved this back into the component? I thought the intent was not to reach for helpers directly from the components if these get an accessor from a context. Don't think one way or the other is best but curious about why the reversal.

Reply:
Yeah makes sense to move this function back to the `OnboardingContext` and just import it!